ol3-viewer/src/ome/ol3/globals.js

```diff
@@ -428,6 +430,18 @@ ome.ol3.defaultInteractions = function() {
                     "ref": new Constructor(V['options']),
                     "defaults" : V["defaults"]
                 };
+                // For some reason, calling new ol.interaction.DoubleClickZoom({"delta": 10})
+                // here, the 'delta' option is ignored. But if we create a defaults collection
+                // the same class is created and the {'delta': zoomDelta} is not ignored.
+                // We use altShiftDragRotate: false so that doubleClickZoom should be
+                // the first item in the ol.Collection.
+                if (K === "doubleClickZoom") {
```

Comment:
@will-moore I wanted to just quickly point you towards what's a bit of deceptive behavior from the closure compiler but in this case you should be able to achieve the same without this special if but specifying the properties you want to set as a string, e.g `"options": {"duration" : 1000, "delta": 10}` 
The reason I say this is that I suspect very strongly that the closure compiler mangles the property which it does not do if supplied in "string notation". It never touches strings.

Now I don't think this is a major problem any more but I refrained from using ECMA2015 constructs in the ol3 code such as "let". As is the ol3 viewer gets built by the closure compiler and is then used as a import and as such does not go through the babel transpiler. Problems nowadays should be rare but I did encounter one problem in the past using such a feature and IE11 (not 'let' though but one of the ones that got implemented later on...), that's why I'm mentioning it -for completeness sake ;-)

Reply:
Hi @waxenegger, thanks for the input. I played a bit more and things seem to be working better now. When I tried it before with e.g. ```{"duration" : 1000, "delta": 10}``` the ```duration``` was getting recognised but the ```delta``` wasn't.
Anyway, I seem to have it working without the workaround now in #213.